src/unix.rs

```diff
@@ -103,6 +103,17 @@ impl RealtimeThreadSchedulePolicy {
     }
 }
 
+/// Flags for controlling Deadline scheduling behavior.
```

Comment:
Can you please also add a link to any online "man" where this is also documented? Just in case one needs more information.